cfg/config.yaml

```diff
@@ -78,15 +78,26 @@ node:
     - kubernetes
 
   kubernetes:
-    defaultconf: /etc/kubernetes/config    
+    confs:
+      - "/var/lib/kubelet/kubeconfig"
```

Comment:
kubernetesconf is only referred to in the test config file for 1.7 so I don't think this change can have any effect, so I'd prefer not to make it. (in fact I think we should be removing this section altogether and I've raised #384 for it).  This does look like a kubelet file anyway, and I think the effect you're looking for has likely been achieved by adding it into the kubelet section below